src/tool/subcommands/api_cmd.rs

```diff
@@ -831,6 +831,11 @@ fn state_tests_with_tipset<DB: Blockstore>(
                 block.miner_address,
                 tipset.key().into(),
             ))?),
+            RpcTest::identity(StateGetClaim::request((
+                block.miner_address,
+                0,
```

Comment:
Is `0` usually a valid claim id or will this test always return `null`?

Reply:
We actually have to implement `StateGetClaims` to be able to test that one. I don't think there's another good way to supply a valid claim_id, as it depends on the miner. I will add a comment.